open-class/docs/classes/assignment-branch-logic.md

```diff
@@ -10,244 +10,187 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+Well, without user research, you don't really know what good questions would be. You don't have funding or time to do that discovery work, though, but you don't care that much about those strangers and it's not like they have other options, so you'll start by making a guess and worry about it later.
 
-Let's look at line 3 of our sample:
+Do they even like parties? That seems like a place to start.
 
 ```yaml
-yesno: over_eighteen
+---
+id: does_like_parties
+question: |
+  Do you like parties?
+fields:
+  - Do you like parties: user_likes_parties
+---
 ```
+<!-- I'd stick with fields for now because those will be used most consistently -->
 
-This is a new specifier: the `yesno` specifier. It always shows the text `yes` and `no` as buttons on the screen. On the right hand side of the : is the variable that will be assigned the value `True` or `False`, depending on which button the user clicks.
-
-### Planning out our variables
-
-It can help to start out by writing a list of the actual variables we will want to ask. This way, we can plan variable names that make sense together.
-
-In Docassemble interviews, it's traditional to use variables names that are:
+Docassemble will show two buttons on this question screen. If the user clicks 'Yes', `user_likes_parties` will have a value of `True`. If they click 'No', it will have a value of `False`. We can use that to help our program decide what to do. [Read more about yes/no fields](https://docassemble.org/docs/fields.html#fields%20yesno).
 
-1. Short
-1. Descriptive (without cryptic abbreviations)
-1. Lower-case
-1. Use _ to separate multiple words when needed
+### If this, then that
 
-We are going to need true/false variables and an ending screen. Let's name them as follows:
+We'll use what's called an `if` statement. It tells our program, 'if the situation is this, then do that.'
 
-variable name | meaning
---------------|---------
-go_to_party | A screen that tells the person to go to the party
-dont_go_to_party | A screen that says not to go to the party
-likes_parties | Do you like parties?
-wants_to_meet | Do you want to meet someone there?
-big_assignment | Do you have a big assignment due?
-
-### Writing our first question blocks
-
-Let's start with writing the ending blocks, which are the simplest. Remember, we will use the `event` specifier for these because they are ending screens.
-
-```yaml
+``` yaml
+---
+id: flow
+mandatory: True
+code: |
+  if ( user_likes_parties ):
+    go_to_party
+  else:
+    dont_go_to_party
 ---
-event: go_to_party
-question: |
-  You should go to the party!
-subquestion: |
-  [FILE https://media.giphy.com/media/l0MYt5jPR6QX5pnqM/giphy.gif]
 ```
 
-[![](https://media.giphy.com/media/l0MYt5jPR6QX5pnqM/giphy.gif)]
+### More branches!
+
+So that's not really all that matters, right? For example, do they have a ride or would they have to walk?
 
 ```yaml
 ---
-event: dont_go_to_party
+id: has_a_ride
 question: |
-  You should just go to sleep
```

Comment:
Note: Ripe for a deep discussion about psychological and social issues! Is there something you need to tell me? Seriously, though, it is interesting how our perspectives come out in our code. It's a good thing for a developer to learn, though maybe more explicitly.